_docs/guides/best-practices.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Best Practices 
```

Comment:
I do think though that it is a great improvement over what we had before, hence approved.